symengine/lib/symengine_wrapper.pyx

```diff
@@ -778,10 +790,38 @@ cdef class Basic(object):
         return False
 
     @property
+    def is_integer(self):
+        return False
+
+    @property
+    def is_finite(self):
+        return None
+
+    @property
     def is_Derivative(self):
         return False
 
     @property
+    def is_AlgebraicNumber(self):
+        return False
```

Comment:
If you are defining it here, where's the function that's supposed to return `True`?